src/pages/home/report/ReportActionsView.js

```diff
@@ -76,29 +76,28 @@ class ReportActionsView extends React.Component {
         // Helper variable that keeps track of the unread action count before it updates to zero
         this.unreadActionCount = 0;
 
+        // Todo: this should come from props and depend on isDrawerOutOfTheWay
         // Helper variable that prevents the unread indicator to show up for new messages
         // received while the report is still active
         this.shouldShowUnreadActionIndicator = true;
 
         this.state = {
             isLoadingMoreChats: false,
         };
+
+        this.updateSortedReportActions(props.reportActions);
     }
 
     componentDidMount() {
         AppState.addEventListener('change', this.onVisibilityChange);
         subscribeToReportTypingEvents(this.props.reportID);
         this.keyboardEvent = Keyboard.addListener('keyboardDidShow', this.scrollToListBottom);
         this.recordMaxAction();
-        fetchActions(this.props.reportID);
     }
 
     shouldComponentUpdate(nextProps, nextState) {
-        if (nextProps.reportID !== this.props.reportID) {
-            return true;
-        }
```

Comment:
No longer needed due to re-mounting

Reply:
I'm probably missing something obvious here, but why does this component mount and unmount now whereas before it did not?

Reply:
Answered here: https://github.com/Expensify/Expensify.cash/pull/2221#issuecomment-815943686